github-management/opening-a-request.md

```diff
@@ -0,0 +1,33 @@
+# Opening a issue for support with GitHub
+
+## GitHub issues
+
+If you need help with the following:
+- Permissions issues
+- Organization membership
+- Third-party integrations
+- Webhooks
+- Repository creation/migration
+- Repository archival
+- Other repository and configuration issues
+
+Please open an issue against the [kubernetes/org] repository describing your
+issue. If your request is urgent, please escalate to **[@kubernetes/owners]**.
+
+## Bot/Automation issues
+
+If you need help with the following:
+- Bot configuration
+- Automatic merging
+- Label issues
+- Automation feedback and feature requests
+
+Please open an issue against the [kubernetes/test-infra] repository describing
+your issue. If your request is urgent, please escalate to the
+[test-infra on-call] or reach out to #testing-ops on Slack.
```

Comment:
Can you backtick `#testing-ops` so it is more emphasized? 

https://github.com/kubernetes/community/blob/9d38dabf5dc53706731f3f0680e723271d86c964/github-management/opening-a-request.md

Reply:
Done.